src/components/WikiTags/Report/Report.js

```diff
@@ -32,11 +79,7 @@ class OlabReportTag extends React.Component {
         );
       }
 
-      return (
-        <>
-          <b>[[REPORT]]</b>
-        </>
-      );
+      return (<Report {...this.props} report={report} />);
```

Comment:
Could you please tweak this to use only class components?  And one component per file.